src/main.cpp

```diff
@@ -490,10 +490,6 @@ int main(int argc, char *argv[])
   if (!clang.parse(driver.root_, bpftrace, extra_flags))
     return 1;
 
-  err = driver.parse();
```

Comment:
I think this second parse is needed to handle `#defines` (but a comment explaining that would be good if I'm right)